index.html

```diff
@@ -10856,12 +10856,31 @@ <h3>Graph Containers</h3>
 </section>
 </section>
 
-<section class="normative"><h2>Interpreting JSON as JSON-LD</h2>
+<section class="normative"><h2>Modifying Behavior with Link Relationships</h2>
+  <p>Certain aspects of JSON-LD processing can be modified using
+    <dfn data-cite="RFC8288#section-3" data-no-xref>HTTP Link Headers</dfn> [[RFC8288]].
+    These can be used when retrieving resources that are not, themselves, JSON-LD,
+    but can be interpreted as JSON-LD by using information in a
+    <a href="https://en.wikipedia.org/wiki/Link_relation" data-no-xref>Link Relation</a>.</p>
+
+  <p>When processing normal JSON documents, a link relation can be specified using
+    the <a>HTTP Link Header</a>
+    returned when fetching a remote document, as described in <a href="#interpreting-json-as-json-ld" class="sectionRef"></a>.</p>
+
+  <p class="changed">In other cases, a resource may be returned using a representation that cannot easily be interpreted
+    as JSON-LD. Normally, <a href="https://en.wikipedia.org/wiki/Content_negotiation">HTTP content negotiation</a>
+    would be used to allow a client to specify a preference for JSON-LD over another representation,
+    but in certain situations, it is not possible for a server to respond appropriately to such requests.
```

Comment:
```suggestion
    but in certain situations, it is not possible or practical for a server to respond appropriately to such requests.
```

Reply:
Fixed in f1f4efe0c995b9ab901ed19cffdf7d286bd3961b.